engine/security/src/main/scala/pl/touk/nussknacker/ui/security/basicauth/BasicAuthenticatorFactory.scala

```diff
@@ -6,7 +6,9 @@ import com.typesafe.config.Config
 import pl.touk.nussknacker.ui.security.api.AuthenticatorFactory.{AuthenticatorData, LoggedUserAuth}
 import pl.touk.nussknacker.ui.security.api.{AuthenticatorFactory, DefaultAuthenticationConfiguration}
 
-class BasicAuthenticatorFactory extends AuthenticatorFactory with Directives {
+import scala.concurrent.ExecutionContext
+
+class BasicAuthenticatorFactory(implicit ec: ExecutionContext) extends AuthenticatorFactory with Directives {
```

Comment:
I think we don't need it, see comment in BasicHttpAuthenticator

Reply:
Removed